readme.md

```diff
@@ -3,15 +3,15 @@
 # Skipper
 
 Skipper is an HTTP router that acts as a reverse proxy (with support for flexible route definitions), alters
-requests, and responses with filters. It:
+requests, and responds with filters. It:
```

Comment:
I meant by 'in between' as between the request processing starts and the response is returned, but actually, it isn't technically precise either way. So, I think the simplest would be to simply leave the 'in between' out:

Skipper is an HTTP router that acts as a reverse proxy, mapping incoming requests to multiple HTTP backend services based on routes selected by the request attributes. Both the requests and responses can be augmented by a filter chain defined individually for each route.